src/SimpleExcelReader.php

```diff
@@ -312,17 +307,37 @@ protected function getValueFromRow(Row $row): array
     protected function getSheet(): SheetInterface
     {
         $this->reader->open($this->path);
+        if ($this->searchSheetByName) {
+            $sheet = $this->getActiveSheetByName();
+        } else {
```

Comment:
That already looks way better! 

In general, I try to avoid  `else` whenever I can as it tends to make code less readable. Let's use a ternary here instead of an `else` statement.